platform.target

```diff
@@ -6,6 +6,8 @@
 			<repository location="http://download.eclipse.org/releases/2020-03"/>
 			<unit id="org.eclipse.platform.ide" version="0.0.0"/>
 			<unit id="org.eclipse.jdt.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.swtbot.eclipse.feature.group" version="0.0.0"/>
```

Comment:
Interesting, I was not aware SWTBot was part of the Eclipse release update site...